### DISCUSSION TITLE
[openwrt 23.05] iptables-nft Couldn't load match `string':No such file or directory

### DISCUSSION THREAD

**Discussion by liangcz** [v2rayA/v2rayA#1476] [Q&A]
hi guys, i'm trying to start v2rayA on openwrt, but have some error occured.

this error maybe caused by iptables-nft -m  option, but i don't know how to fix it. 

error log:
[W] [processManager.go:75] DropSpoofing can't be enable: ExecCommands: iptables-nft -w 2 -A DROP_SPOOFING -p udp --sport 53 -m string --algo bm --hex-string "|00047f|" --from 60 --to 180 -j DROP iptables v1.8.8 (nf_tables): Couldn't load match `string':No such file or directory

**Comment by liangcz** [comment]
fix this problem by this command
opkg install iptables-mod-filter

**Reply by jingmian** [reply]
@liangcz very good
